Templates/Common-Backend-Scripts/dbbBuild.sh

```diff
@@ -370,15 +370,11 @@ validateOptions() {
   esac
 
   if [ -z "${dbbMetadataStoreJdbcId}" ]; then
-    rc=8
-    ERRMSG=$PGM": [ERROR] DBB database user is required. rc="$rc
-    echo $ERRMSG
+    echo $PGM": [WARNING] Db2 JDBC User not set. The usage of the Db2 DBB Metadatastore is recommended."
   fi
 
   if [ -z "${dbbMetadataStoreJdbcPwdFile}" ]; then
-    rc=8
-    ERRMSG=$PGM": [ERROR] DBB database password file is required. rc="$rc
-    echo $ERRMSG
+    echo $PGM": [WARNING] Db2 JDBC Password file not set. The usage of the Db2 DBB Metadatastore is recommended."
```

Comment:
Could we rephrase the last sentence to:
For the DBB metadata database, the use of Db2 (z/OS or LUW) is recommended.